### PR TITLE
doc: update the Seastar Perftune page

### PR DESCRIPTION
**Pull request by annastuchlik**
This commit updates the description of perftune.py. It is based on the information in the reported issue (below), the contents of help for perftune.py, and the input from @vladzcloudius.

Fixes https://github.com/scylladb/scylladb/issues/14233

### CODE REVIEW

**Comment by annastuchlik** [comment]
Preview
image
image
image
image

**Comment by annastuchlik** [comment]
@vladzcloudius Please review this PR.

**Comment by annastuchlik** [comment]
@asias @syuu1228 Could you review this PR, too?
Also, could you explain to @tzach and me how the user can run the script? We thought it was run from scylla_setup, but it seems it's run by the scylla-server service. Is that correct?

What are the steps the user needs to take to run perftune.py?

File: docs/operating-scylla/admin-tools/perftune.rst [hunk 1]
```diff
@@ -1,51 +1,90 @@
...
+     - Specifies the components to configure. You can use it more than once to specify 
+       multiple values. The available values are:
 
-Default values:
-
-* --nic NIC: eth0
-* --cpu-mask MASK: all available cores mask
-* --tune-clock: false
+         * ``net`` - Enables tuning the network.
+         * ``disk`` - Enables tuning the disks.
```
Comment (vladzcloudius) [inline docs/operating-scylla/admin-tools/perftune.rst:82]: It's `disks` not `disk`

**Review by vladzcloudius — CHANGES_REQUESTED** [review]

**Review by vladzcloudius — CHANGES_REQUESTED** [review]

**Review by vladzcloudius — CHANGES_REQUESTED** [review]
On top of my other comments - some examples of perftune.py usage would be nice.

File: docs/operating-scylla/admin-tools/perftune.rst [hunk 2]
```diff
@@ -1,51 +1,90 @@
...
 
-As a result, some of the CPUs may be destined only to handle the IRQs and taken out of the CPU set to be used to run the seastar application ("compute CPU set").
+To ensure that the tuning persists after a host reboot, we recommend creating a systemd unit 
+that will run on host startup and re-tune the hosts.
 
+perftune.yaml
+------------------
 
-perftune.py [options]
+The ``perftune.yaml`` file is generated by the ``perftune.py`` script.The script determines
```
[the remark this answers is not in the document]
Comment (annastuchlik) [reply docs/operating-scylla/admin-tools/perftune.rst:23]: I'll replace it with the information that perfrune.yaml is the output file of perftune.py (with the --dump-options-file enabled).
Would that be sufficient?

**Comment by annastuchlik** [comment]
@vladzcloudius @asias @syuu1228 Cloud you review this PR? I've applied the suggestions from @vladzcloudius.

@vladzcloudius If you provide some real-life examples, I'll be happy to add them. Alternatively, we can merge this PR to have the page improved asap, and add the examples later.

**Review by vladzcloudius — APPROVED** [review]
I think it looks good to merge.

I agree about examples - let's merge this and then work on a follow up.

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please merge.

**Pull request closed without merging**